release-notes/3.0/preview/api-diff/preview7/3.0-preview7_System.Collections.Generic.md

```diff
@@ -0,0 +1,15 @@
+# System.Collections.Generic
+
+``` diff
+ namespace System.Collections.Generic {
+-    public interface IDictionary<TKey, TValue> : ICollection<KeyValuePair<TKey, TValue>>, IEnumerable, IEnumerable<KeyValuePair<TKey, TValue>> where TKey : object
++    public interface IDictionary<TKey, TValue> : ICollection<KeyValuePair<TKey, TValue>>, IEnumerable, IEnumerable<KeyValuePair<TKey, TValue>>
+-    public interface IReadOnlyDictionary<TKey, TValue> : IEnumerable, IEnumerable<KeyValuePair<TKey, TValue>>, IReadOnlyCollection<KeyValuePair<TKey, TValue>> where TKey : object
++    public interface IReadOnlyDictionary<TKey, TValue> : IEnumerable, IEnumerable<KeyValuePair<TKey, TValue>>, IReadOnlyCollection<KeyValuePair<TKey, TValue>>
+     public class SortedList<TKey, TValue> : ICollection, ICollection<KeyValuePair<TKey, TValue>>, IDictionary, IDictionary<TKey, TValue>, IEnumerable, IEnumerable<KeyValuePair<TKey, TValue>>, IReadOnlyCollection<KeyValuePair<TKey, TValue>>, IReadOnlyDictionary<TKey, TValue> {
+-        void System.Collections.ICollection.CopyTo(Array array, int arrayIndex);
++        void System.Collections.ICollection.CopyTo(Array array, int index);
```

Comment:
Will need to send a PR to fix this, it seems like it slipped in one of my nullable PRs: 
https://github.com/dotnet/corefx/commit/cc8137ede09cdf16811767a6fb262fb498d85e67#diff-9af4fbb6cc264756bf83c6b96c0144b5R569

Reply:
Does this need to be updated now? Or is the fix for preview8?